service/controller/key/key.go

```diff
@@ -52,7 +52,7 @@ const (
 	CertificateEncryptionKeyName   = "encryptionkey"
 	CertificateEncryptionIVName    = "encryptioniv"
 
-	CoreosVersion = "2303.4.0"
+	ContainerLinuxComponentName = "containerlinux"
```

Comment:
Maybe for the future but should that key not be operator system?

Reply:
True when we move away from `containerlinux` eventually - I didn't want to overdo it here :D